lib/__tests__/fixtures/config-extending-with-object.mjs

```diff
@@ -0,0 +1,5 @@
+import configBlockNoEmpty from './config-block-no-empty.mjs';
+
+export default {
+	extends: configBlockNoEmpty,
```

Comment:
nit

You could import `lib/__tests__/fixtures/config-block-no-empty.json` instead.
i.e. you don't need to create `lib/__tests__/fixtures/config-block-no-empty.mjs`

Reply:
Importing JSON with pure ESM currently needs Import assertions, so I think it's fine as it is.